mteb/tasks/Retrieval/multilingual/BelebeleRetrieval.py

```diff
@@ -0,0 +1,213 @@
+    "srp": ["srp-Cyrl"],
+    "ssw": ["ssw-Latn"],
+    "sun": ["sun-Latn"],
+    "swe": ["swe-Latn"],
+    "swh": ["swh-Latn"],
+    "tam": ["tam-Taml"],
+    "tel": ["tel-Telu"],
+    "tgk": ["tgk-Cyrl"],
+    "tgl": ["tgl-Latn"],
+    "tha": ["tha-Thai"],
+    "tir": ["tir-Ethi"],
+    "tsn": ["tsn-Latn"],
+    "tso": ["tso-Latn"],
+    "tur": ["tur-Latn"],
+    "ukr": ["ukr-Cyrl"],
+    "urd": ["urd-Arab"],
+    "uzn": ["uzn-Latn"],
+    "vie": ["vie-Latn"],
+    "war": ["war-Latn"],
+    "wol": ["wol-Latn"],
+    "xho": ["xho-Latn"],
+    "yor": ["yor-Latn"],
+    "zho": ["zho-Hans"],
+    "zsm": ["zsm-Latn"],
+    "zul": ["zul-Latn"],
+}
+
+
+class BelebeleRetrieval(MultilingualTask, AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="BelebeleRetrieval",
+        dataset={
+            "path": "facebook/belebele",
+            "revision": "75b399394a9803252cfec289d103de462763db7c",
+        },
+        description=(
+            "Belebele is a multiple-choice machine reading comprehension (MRC) dataset spanning 115 distinct languages."
+        ),
+        type="Retrieval",
+        category="s2p",
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=_LANGS,
+        reference="https://arxiv.org/abs/2308.16884",
+        main_score="ndcg_at_20",
+        license="CC-BY-SA-4.0",
+        domains=["Web", "News"],
+        text_creation="created",
+        n_samples={_EVAL_SPLIT: 900},
+        date=("2023-08-31", "2023-08-31"),
+        form=["written"],
+        task_subtypes=["Question answering"],
+        socioeconomic_status="mixed",
+        annotations_creators="expert-annotated",
+        dialect=[],
+        avg_character_length={_EVAL_SPLIT: 329},
+        bibtex_citation="""@article{bandarkar2023belebele,
+      title={The Belebele Benchmark: a Parallel Reading Comprehension Dataset in 122 Language Variants}, 
+      author={Lucas Bandarkar and Davis Liang and Benjamin Muller and Mikel Artetxe and Satya Narayan Shukla and Donald Husa and Naman Goyal and Abhinandan Krishnan and Luke Zettlemoyer and Madian Khabsa},
+      year={2023},
+      journal={arXiv preprint arXiv:2308.16884}
+}""",
+    )
+
+    def load_data(self, **kwargs) -> None:
+        if self.data_loaded:
+            return
+
+        self.dataset = load_dataset(**self.metadata_dict["dataset"])
+
+        self.queries = {lang: {_EVAL_SPLIT: {}} for lang in self.langs}
+        self.corpus = {lang: {_EVAL_SPLIT: {}} for lang in self.langs}
+        self.relevant_docs = {lang: {_EVAL_SPLIT: {}} for lang in self.langs}
+
+        for lang in self.langs:
+            belebele_lang = _LANGS[lang][0].replace("-", "_")
+            ds = self.dataset[belebele_lang]
+
+            question_ids = {
+                question: _id for _id, question in enumerate(set(ds["question"]))
+            }
+            context_ids = {
+                passage: _id for _id, passage in enumerate(set(ds["flores_passage"]))
+            }
+            answers = [
```

Comment:
I'm not sure we need the answers, the corpus containing all the text should be enough for the retrieval task, as the goal is to retrieve the right docs for a question.

Reply:
[GermanQuAD](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/Retrieval/deu/GermanQuADRetrieval.py) and [FQuAD](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/Retrieval/fra/FQuADRetrieval.py) don't use answers, we'll check again these 2 tasks. I think you can remove answers yes.